packages/nimble-components/src/rich-text/viewer/tests/rich-text-viewer.spec.ts

```diff
@@ -2,13 +2,43 @@ import { html } from '@microsoft/fast-element';
 import { RichTextViewer, richTextViewerTag } from '..';
 import { fixture, type Fixture } from '../../../utilities/tests/fixture';
 import { RichTextViewerPageObject } from '../testing/rich-text-viewer.pageobject';
+import { anchorTag } from '../../../anchor';
```

Comment:
We should probably also add test cases using the pattern in #1633

Reply:
I think the PR you mentioned above is testing strategy PR for serializer, In that what pattern you are meaning @jattasNI ?

Reply:
Sorry, left that comment in a rush at the end of the day. We will want to write interaction tests like that when we add @mention support to the rich text editor but it's not needed for the rich text viewer.